lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -7,25 +7,21 @@ define((require, exports, module) ->
         placement: "bottom"
         trigger: "manual"
         title: "Formula"
-        container: "#editor"
+        container: editor.container
       }
 
-      show: (jqPopoverContainer, content) ->
-        cursorPosition = $("textarea.ace_text-input").position()
-        jqPopoverContainer.css({
-          top: "#{cursorPosition.top + 24}px"
-          left: "#{cursorPosition.left}px"
-        })
+      show: (jqPopoverContainer, content, position) ->
+        jqPopoverContainer.css(position)
         popoverHandler.options.content = content
         jqPopoverContainer.popover(popoverHandler.options)
         jqPopoverContainer.popover("show")
         return
 
-      hide: (jqPopoverContainer) ->
+      destroy: (jqPopoverContainer) ->
         jqPopoverContainer.popover("destroy")
 
-      isVisible: (jqPopoverElement) ->
-        jqPopoverElement.children(".popover").is(":visible")
+      isVisible: (jqPopoverContainer) ->
+        jqPopoverContainer.data().popover.tip().hasClass("in")
```

Comment:
But they are not. See, the way it was before, which is:
```
isVisible: (jqPopoverElement) ->
  jqPopoverElement.children(".popover").is(":visible")
```

`jqPopoverElement` had to be the actual popover element (**not** `$("#formula")`). So, before calling `isVisible`, I had to get the actual popover element somehow. 

So, I could inspect the page and then pick it manually using jQuery, like `$(".popover-content")`. But in this case, what if there are other popovers? From the popover itself, there is no way to find out its origin. So there is a risk to pick the wrong one.

The only way I found to have the right popover element 100% of the time is the way I used here. So I decided to go with it.

By the way, so far I didn't use this method once in my code, so one of the options is to just remove it altogether :)